### ISSUE TITLE
Unexpected behavior of snapToAngle

### ISSUE THREAD

**Issue by maxmitti** [svgdotjs/svg.resize.js#48]
I'm not sure if it is intended behavior or an issue, but for my use case i expected snapToAngle to behave a bit differently:
- The snapping happens relative to the start point of the currently active change of rotation. I'd expect it to snap to absolute angles, like e.g. Inkscape does too. This allows to easily "reset" some arbitrary rotation back to a sane angle and also align multiple different elements to the same angle.
- The snapped angle is rounded downwards, instead of rounded to the nearest possible angle, which makes using it feel a bit strange. It snaps away as soon as the currently snapped angle is reached with the cursor.

I "fixed" both in my working copy. If there is interest into it i can make a pull request with the changes tomorrow.

**Comment by Fuzzyma** [comment]
Yes please create a PR. Thats sounds like something we want! I didnt put too much thoughts into that feature.
